lib/dor/text_extraction/abbyy/results.rb

```diff
@@ -5,27 +5,35 @@ module TextExtraction
     module Abbyy
       # Parses .xml.result.xml file that gets created when Abbyy sees a XMLTICKET
       class Results
-        attr_reader :result_path, :druid, :run_index
+        attr_reader :result_path, :druid, :run_index, :software_name, :software_version
 
         # Naming convention for result XML files:
         # ab123cd4567.xml.result.xml
         # ab123cd4567.xml.0001.result.xml
         RESULT_FILE_PATTERN = /(?<druid>\w+)\.xml(?:\.(?<run_index>\d+))?\.result\.xml/
 
+        # Class variables so we can use the last value if we don't have ALTO to check
+        @last_software_name = nil
+        @last_software_version = nil
```

Comment:
this stuff gets thorny for me to understand, but I _thought_ that declaring them with the single `@` in the class body (instead of in `#initialize`) makes them "class instance" variables, while `@@` would make them bona-fide "class variables". rubocop does not like the latter.

open question as to whether this is the right strategy to "remember" this information, though...it just seemed like the simplest 🤷🏻

Reply:
oh!  good point, you might be right, i'm not sure about this either.

at any rate, it seems to work, and should be thread safe enough for us, so all good, i think.